FiveOhFirstDataCore/Pages/Index.razor

```diff
@@ -1,9 +1,32 @@
-﻿@page "/"
+@inject IAuthorizationService _auth
+@page "/"
 
-<h1>Hello there!</h1>
-
-General Kenobi.
+<AuthorizeView>
+ <Authorized>
+  <h1>Welcome back, @CurrentUser?.NickName!!</h1>
+ </Authorized>
+ <NotAuthorized>
+     <h1>Welcome Guest!</h1>
+ </NotAuthorized>
+</AuthorizeView>
+// Future Update, The Line Below will be setup for random quotes API
```

Comment:
This is not a comment on a Razor page. Either use HTML comments or Blazor comments (`<!-- -->` or `@* *@`)

Reply:
I blame php - it has me set up with //